internal/common/roles/judge.go

```diff
@@ -8,13 +8,41 @@ import (
 // Judge is an interface that is implemented by baseJudge but can also be
 // optionally implemented by individual islands.
 
+// EvaluationReturn is a data-structure allowing clients to return which rules they've evaluated and the results
 type EvaluationReturn struct {
 	Rules       []rules.RuleMatrix
 	Evaluations []bool
 }
+
+// Sanction is a data-structure that represents a sanction on an agent, including how long it has left
+type Sanction struct {
+	ClientID     shared.ClientID
+	SanctionTier IIGOSanctionTier
+	TurnsLeft    int
+}
+
+// IIGOSanctionScore provides typed integer score for each island
+type IIGOSanctionScore int
+
+// IIGOSanctionTier provides typed integer tiers for sanctions
+type IIGOSanctionTier int
+
+// Provides enumerated tiers for IIGO sanctions
+const (
+	SanctionTier1 IIGOSanctionTier = iota
+	SanctionTier2
+	SanctionTier3
+	SanctionTier4
+	SanctionTier5
+	None
```

Comment:
`None` will be a global const in this package. Let's name it something more specific to the type

Reply:
Agreed and changed